### ISSUE TITLE
SQL error

### ISSUE THREAD

**Issue by SpunkyB21** [ESX-Org/es_extended#235]
![es_extended error](https://user-images.githubusercontent.com/49198112/55405841-e7f29b80-5528-11e9-9fb5-52a87a4b034e.jpg)
No matter what I do, this error always comes up when I import the es_extended.sql into my database, even if the database is completely empty.

**Comment by schwim0341** [comment]
If the database is completely empty then the SQL file you're trying to import is trying to create the column name twice.  Remove that add column line from the import file since it's already in the database and see if that solves your problem when you try to run the file on a completely empty db like you say you're trying to do.

**Comment by SpunkyB21** [comment]
By "empty" I mean a freshly created database, where nothing has been created or imported, other than this file. In other words, I create the database in phpMyAdmin, and then only import es_extended.sql into it. But I'll try what you suggested. Thanks!

**Comment by schwim0341** [comment]
Then it seems very clear to me that you're adding columns that already exist.  You're going to find that most, if not all, of those columns already exist. in a fresh ESX install.

**Comment by SpunkyB21** [comment]
That's just it, I'm not adding anything LOL The columns are already in the SQL file when I import it, but I was just going through it using Notepad ++ and whenever I'd remove a line and import the file into the DB again, it would bring up an error on the next line, and then it started with the tables in the file. So I just deleted everything in the SQL file except where it says at the top "use essentialmode" and got no errors when importing the file.

![es_extended original](https://user-images.githubusercontent.com/49198112/55408005-22f6ce00-552d-11e9-8532-05f86ba8f8a9.jpg)
This is what the original SQL file looks like in Notepad ++

![es_extended edit](https://user-images.githubusercontent.com/49198112/55408033-34d87100-552d-11e9-90f9-20f4dc1bc891.jpg)
This is what it looks like now since I modified it.

I'm getting no errors when I start my server now, whereas before I had this one:
[MySQL] [Slow Query Warning] [es_extended] [1018ms] SELECT * FROM items
[ERROR] [MySQL] [es_extended] An error happens on MySQL for query "[object Object]": connect ECONNREFUSED 127.0.0.1:3306
Error: (node:508) UnhandledPromiseRejectionWarning: Error: connect ECONNREFUSED 127.0.0.1:3306
at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1191:14)
--------------------
at Protocol._enqueue (mysql-async.js:5807:48)
at Protocol.handshake (mysql-async.js:5714:23)
at PoolConnection.connect (mysql-async.js:720:18)
at Pool.getConnection (mysql-async.js:4577:16)
at Pool.query (mysql-async.js:4731:8)
at Promise (mysql-async.js:4970:12)
at new Promise (<anonymous>)
at execute (mysql-async.js:4967:26)
at Object.global.exports [as callback] (mysql-async.js:4993:5)
at citizen:/scripting/v8/main.js:96:41
Error: (node:508) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
Error: (node:508) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

Now I'm just wondering what the point was in having all that extra code in the SQL file if it's not really needed in the first place and only causes problems (at least for me). I'm totally new to all of this, so I'm slowly learning, or trying to.
